File: contest/a_loot_sharing/sol/sol.cs
using System;
					
public class Program
{
	public static void Main()
	{
		int b = Convert.ToInt32(Console.ReadLine());
		int t = Convert.ToInt32(Console.ReadLine());
		int n = Convert.ToInt32(Console.ReadLine());
		
		if(2*b+3*t <= n){
			Console.WriteLine("LOOT!\n");
		}else{
			Console.WriteLine("RHUM!\n");
		}
	}
}

File: contest/a_loot_sharing/sol/sol.cpp
#include <iostream>
using namespace std;

int main() {
    long long int b, t, n;
    cin >> b >> t >> n;
    if(2*b + 3*t <= n)
        cout << "LOOT!" << endl;
    else
        cout << "RHUM!" << endl;
    return 0;
}